runtime/src/main/scala/akka/grpc/GrpcProtocol.scala

```diff
@@ -21,27 +22,34 @@ import scala.util.Try
 trait GrpcProtocol {
```

Comment:
Shouldn't the whole class be `@DoNotInherit` and/or `@InternalApi` when it can only be implemented internally?

Reply:
Not sure yet, I suspect 'GrpcProtocol' will show up in public API's (even if the specific implementations don't and users can't provide their own) - but maybe it can become internal entirely.